demos/toolbar/index.html

```diff
@@ -33,9 +33,24 @@
       <h2>Normal Toolbar <button type="button" onclick="toggleRTL(0)">Toggle RTL</button></h2>
       <p><a href="./default-toolbar.html" target="_blank">View in separate window</a></p>
       <iframe class="demo-container" src="./default-toolbar.html"></iframe>
-      <h2>Fixed Toolbar <button type="button" onclick="toggleRTL(1)">Toggle RTL</button></h2>
+      <h2>Waterfall Toolbar<button type="button" onclick="toggleRTL(1)">Toggle RTL</button></h2>
       <p><a href="./fixed-top-toolbar.html" target="_blank">View in separate window</a></p>
       <iframe class="demo-container" src="./fixed-top-toolbar.html"></iframe>
+      <h2>Waterfall Toolbar (CSS Only)<button type="button" onclick="toggleRTL(2)">Toggle RTL</button></h2>
```

Comment:
(This comment got folded below so I decide to move it upward.)

I feel a bit confused about the relationship between waterfall toolbar and fixed toolbar.
In my understanding: fixed toolbar = waterfall toolbar without JS / css-only fall back for waterfall toolbar.

If that's not our intention, we can either break it into 2 sections:
- Fixed toolbar
- Waterfall toolbar

Or we can organize words differently here.